llama-index-integrations/multi_modal_llms/llama-index-multi-modal-llms-openai/pyproject.toml

```diff
@@ -27,7 +27,7 @@ exclude = ["**/BUILD"]
 license = "MIT"
 name = "llama-index-multi-modal-llms-openai"
 readme = "README.md"
-version = "0.2.1"
+version = "0.2.2"
 
 [tool.poetry.dependencies]
```

Comment:
Probably this will require dependending on `llama-index-openai-llm = "^0.2.10" right?

Reply:
Good catch, updated